.ci/test-install-docs.sh

```diff
@@ -11,18 +11,48 @@ set -e
 export GOPATH=${GOPATH:-${HOME}/go}
 
 typeset -r docker_image="busybox"
+typeset -r kata_project_url="github.com/kata-containers"
+typeset -r test_repo="${kata_project_url}/tests"
+typeset -r test_repo_url="https://${test_repo}"
+typeset -r test_repo_dir="${GOPATH}/src/${test_repo}"
+typeset -r kata_project_dir="${GOPATH}/src/${kata_project_url}"
+
+typeset -r mgr="${test_repo_dir}/cmd/kata-manager/kata-manager.sh"
+
+die()
+{
+	local msg="$*"
+	echo >&2 "ERROR: $msg"
+	exit 1
+}
 
 info()
 {
 	local msg="$*"
 	echo "INFO: $msg"
 }
 
+# Grab a copy of the tests repository
+get_tests_repo()
+{
+	[ -d "${test_repo_dir}" ] && return
```

Comment:
I doubt we need to, but if it already exists, should we do a pull/fetch on it to make sure it is upto date?

Reply:
Good question but this test is only designed to run under the CI where everything is fresh.

Reply:
I tend to agree with Graham, sometimes I use CI scripts locally, not required in this PR we can let the user update test repository manually.